Assets/00_Content/Scripts/Interactables/Table.cs

```diff
@@ -76,23 +76,20 @@ public class Table : Interactable {
 		}
 
 		private void StartRepairing(GameObject player, RepairTool tool) {
-			if (Tavern.Instance != null && RoundController.Instance != null)
-				Tavern.Instance.SpendsMoney(RoundController.Instance.CurrentDifficulty.tableRepairCost);
+			if (Tavern.Instance != null && RoundController.Instance != null) {
 
-			float repairTime = RoundController.Instance != null
-				? RoundController.Instance.CurrentDifficulty.tableRepairTime
-				: 5f;
-			tool.BeginRepairing(repairTime, transform.position + new Vector3(0, 3f, 0));//event
-			tool.RepairDone += HandleRepairDone;
+				float repairTime = RoundController.Instance != null
+					? RoundController.Instance.CurrentDifficulty.tableRepairTime
+					: 5f;
+				tool.BeginRepairing(repairTime, transform.position + new Vector3(0, 3f, 0)); //event
+				tool.RepairDone += HandleRepairDone;
 
-			player.GetComponent<PlayerMovement>().CanMove = false;
+				player.GetComponent<PlayerMovement>().CanMove = false;
+			}
 		}
```

Comment:
Why did you surround the whole block in the tavern/roundcontroller null check? Couldn't it just be removd?